samples/bq_utils/update_schema.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+#This script helps to perform an upgrade of column 'STATUS' to 'VALIDATION_STATUS' without losing existing data in Results table.
+#It takes backup of the existing biqquery table 'RESULTS' and creates a new table with column name as 'VALIDATION_STATUS'
+
+#PROJECT=pso-kokoro-resources
+DEPRECATED_TABLE=pso_data_validator.results_deprecated #--Deprecated results table with column name 'status'
+CURRENT_TABLE=pso_data_validator.results #--Current results table
```

Comment:
Forgive my ignorance... 

Since the `-bqrh` flag allows the user to specify the output table in `PROJECT_ID.DATASET.TABLE` format... wouldn't it be better to take this an input?

```
./update_schema.sh <current-bq-result-table>
```